pom.xml

```diff
@@ -129,5 +129,10 @@
             <artifactId>Saxon-HE</artifactId>
             <version>9.5.1-5</version>
         </dependency>
+        <dependency>
```

Comment:
This dependency was removed because it is no longer used.  It was causing errors.